services/QuillLMS/spec/workers/snapshots/cache_snapshot_count_worker_spec.rb

```diff
@@ -31,15 +31,11 @@ module Snapshots
       let(:perform) { subject.perform(cache_key, query, user_id, timeframe, school_ids, filters) }
       let(:timeframe_end) { DateTime.now }
       let(:current_timeframe_start) { timeframe_end - 30.days }
-      let(:previous_timeframe_start) { current_timeframe_start - 30.days }
```

Comment:
This change is from https://github.com/empirical-org/Empirical-Core/pull/11138 and does not need to be reviewed here.